lib/OktaAuth.ts

```diff
@@ -569,15 +554,40 @@ class OktaAuthBrowser extends OktaAuthBase implements OktaAuth, SignoutAPI {
   isAuthorizationCodeFlow(): boolean {
     return this.hasResponseType('code');
   }
+
+  // { username, password, (relayState), (context) }
```

Comment:
remove?

Reply:
This method could also be a starting point for interaction code flow